### ISSUE TITLE
Failed to launch on MacOS 11.4, M1 Mac

### ISSUE THREAD

**Issue by tvgold42** [jdah/minecraft-weekend#55]
when running the game with the default arguments it returns

`GLFW error 65544: Cocoa: Failed to find service port for display
error loading shader at res/shaders/basic_texture.vs`

as a short work around, I found that updating GLFW to 3.3.4 and using `-v` in the launch options and recompiling the game managed to make it work.

**Comment by valvhammer** [comment]
Reason:it was made on intel, not M1

**Comment by hdfsyu** [comment]
Please close this issue as the reason is this was compiled for intel.

**Comment by elijahrs-gh** [comment]
Wrong, M1 works just copy and paste these commands into a terminal. 
```
git clone --recurse-submodules https://github.com/jdah/minecraft-weekend.git
cd minecraft-weekend/
brew install libxcursor
make libs
make dirs
make game
make run
```
